### ISSUE TITLE
start-event is generated twice

### ISSUE THREAD

**Issue by SciManJD** [bcakmakoglu/revue-draggable#406]
I have code like this:
```
<div>
        v-draggable="{..., move: onMove, start: onStart}">
...
</div>

...
      methods: {
            onStart() {
                  ...
            }
      }
```

The function onStart() is called twice. I can't find a reason in my code. Is there an explanation why this happens? How can avoid it?

**Comment by bcakmakoglu** [comment]
Thanks for the report, I'll check what's up with that :)

**Comment by bcakmakoglu** [comment]
Will be fixed in the next patch.

**Comment by bcakmakoglu** [comment]
Fixed with `0.5.1`
